src/game_renderer/ViewFrustumCulling.cpp

```diff
@@ -0,0 +1,116 @@
+#include "ViewFrustumCulling.h"
+
+namespace ViewFrustumCulling {
+
+   // Code from CPE-476 lab
+   glm::vec4 * GetViewFrustumPlanes(glm::mat4 P, glm::mat4 V) {
+     glm::vec4 Left, Right, Bottom, Top, Near, Far;
+     glm::vec4 *planes = (glm::vec4 *)malloc(sizeof(glm::vec4) * 6);
```

Comment:
I think a vector would like nicer as well you can even do
`std::shared_ptr<std::vector<glm::vec4>> planes = std::make_shared<std::vector<glm::vec4>>();`
and then do
`planes.reserve(6)`
to make space for the 6 planes
It might also be helpful to have a typedef or struct here that encapsulates the 6 planes and can be nicely addressed something like
`typedef struct planes { glm::vec4 Left; glm::vec4 Right ....`